features/printer.feature

```diff
@@ -5,35 +5,35 @@ Feature: Table printer
 
   Scenario: simple index and valid data
     Given a gherkin table as input
-      | str       | float     | str                 | dict      | list     | OrderedDict              | OrderedDict      |
-      | index_col | float_col | str_col             | dict_col  | list_col | ordered_dict_col         | ordered_dict_col |
-      | egg       | 3.0       | silly walks         | {}        | []       | OrderedDict()            | []               |
-      | spam      | 4.1       | spanish inquisition | {"a": 1}  | ["a"]    | OrderedDict([("a", 1)])  | [("a", 1)]       |
-      | bacon     | 5.2       | dead parrot         | {"a": []} | [1]      | OrderedDict([("a", [])]) | [("a", [])]      |
+      | str       | float     | str                 | dict      | list     | OrderedDict              | OrderedDict        |
+      | index_col | float_col | str_col             | dict_col  | list_col | ordered_dict_col         | ordered_dict_col_2 |
```

Comment:
I already pulled this change in master, I tried to do the exact same change as you so hopefully you won't get conflicts